test/widgets/profile/edit_brith_date_page_test.dart

```diff
@@ -0,0 +1,136 @@
+      await pumpMaterial(tester, EditBirthDatePage(profile));
+      await tester.pump();
+
+      final Finder birthDateFinder = find.text(localeManager.formatDate(profile.birthDate!));
+      final Finder birthDateInputFinder = find.byKey(const Key('birthDateInput'));
+
+      expect(birthDateFinder, findsOneWidget);
+      expect(birthDateInputFinder, findsOneWidget);
+    });
+    testWidgets('change birthDate', (WidgetTester tester) async {
+      await pumpMaterial(tester, EditBirthDatePage(profile));
+      await tester.pump();
+
+      final Finder birthDateFinder = find.text(localeManager.formatDate(profile.birthDate!));
+      final Finder birthDateInputFinder = find.byKey(const Key('birthDateInput'));
+
+      await tester.tap(birthDateInputFinder);
+      await tester.pumpAndSettle();
+      expect(find.byType(DatePickerDialog), findsOneWidget);
+
+      await tester.tap(find.byIcon(Icons.edit));
+      await tester.pumpAndSettle();
+
+      await tester.tap(birthDateFinder.last);
+      final String date = localeManager.formatDate(profile.birthDate!.add(const Duration(days: 5)));
+
+      await tester.enterText(birthDateFinder.last, date);
+      await tester.tap(find.text('OK'));
+      await tester.pumpAndSettle();
+      expect(find.text(date), findsOneWidget);
+    });
+    testWidgets('clear Button', (WidgetTester tester) async {
+      await pumpMaterial(tester, EditBirthDatePage(profile));
+      await tester.pump();
+
+      final Finder birthDateFinder = find.text(localeManager.formatDate(profile.birthDate!));
+
+      expect(birthDateFinder, findsOneWidget);
+
+      final Finder clearButton = find.byKey(const Key('clearButton'));
+      expect(clearButton, findsOneWidget);
+
+      await tester.tap(clearButton);
+      await tester.pump();
+      expect(birthDateFinder, findsNothing);
+    });
+    testWidgets('save Button', (WidgetTester tester) async {
+      await pumpMaterial(tester, ProfilePage.fromProfile(profile));
+      await tester.pump();
+      expect(find.text(profile.age.toString()), findsOneWidget);
+
+      await tester
+          .tap(find.descendant(of: find.byKey(const Key('age')), matching: find.byKey(const Key('editButton'))));
+      await tester.pumpAndSettle();
+      expect(find.byType(EditBirthDatePage), findsOneWidget);
+
+      final Finder clearButton = find.byKey(const Key('clearButton'));
+      await tester.tap(clearButton);
+      await tester.pump();
+
+      final Finder saveButtonFinder = find.byKey(const Key('saveButton'));
+      expect(saveButtonFinder, findsOneWidget);
+      await tester.tap(saveButtonFinder);
+      await tester.pumpAndSettle();
+
+      expect(find.byType(ProfilePage), findsOneWidget);
+
+      verifyRequest(
+        processor,
+        urlMatcher: equals('/rest/v1/profiles?id=eq.1'),
+        methodMatcher: equals('PATCH'),
+        bodyMatcher: equals({'birth_date': null}),
+      ).called(1);
+
+      verifyRequest(processor,
```

Comment:
why does this need to be verified. And can you shorten the url request so that just the relevant part is shown and add methodMatcher: equals('GET') so there is no overlap.

Reply:
I think it's fine to verify the account is reloaded. On the other points, I agree